containers/datalab/content/run.sh

```diff
@@ -104,6 +104,7 @@ if [ -n "${vm_project}" ] && [ "${vm_project}" != "no-project-id" ]; then
    export VM_PROJECT="${vm_project}"
    export VM_NAME=$(curl -s "${compute_metadata_url}/instance/hostname" -H "Metadata-Flavor: Google" | cut -d '.' -f 1)
    export VM_ZONE=$(curl -s "${compute_metadata_url}/instance/zone" -H "Metadata-Flavor: Google" | sed 's/.*zones\///')
+   export DATALAB_SHUTDOWN_COMMAND="gcloud compute instances stop ${VM_NAME} --project ${VM_PROJECT} --zone ${VM_ZONE}"
```

Comment:
By using this environment variable, it is easy for me to set a different shutdown command in my startup.sh that shuts down my local notebook server during development. If you prefer, I can remove this line from this script, and instead add code to idleTimeout.ts that calls getVmInfo, checks to see if we are on a VM, and builds this shutdown command if so.